src/saiga/core/geometry/plane.inl

```diff
@@ -28,6 +28,13 @@ inline HD Plane::Plane(const vec3& p1, const vec3& p2, const vec3& p3)
     d      = dot(p1, this->normal);
 }
 
+inline HD Plane Plane::invert() const
+{
+    vec3 pointOnPlane = getPoint();
+    Plane invPlane(pointOnPlane, -normal);
```

Comment:
Why not just -normal, -d?

Reply:
You are right, change was done while debugging. Will revert that.